### PR TITLE
[CoreSpotlight] Add support for Xcode 14 beta 4.

### PR DESCRIPTION
**Pull request by mandel-macaque**

### CODE REVIEW

**Review by dalexsoto — APPROVED** [review]

File: src/corespotlight.cs [hunk 1]
```diff
@@ -94,9 +94,14 @@ interface CSSearchableIndex {
 		// from interface CSExternalProvider (CSSearchableIndex)
 
 		[Async (ResultTypeName="CSSearchableIndexBundleDataResult")]
-		[NoTV, iOS (16,0), Mac (13,0), MacCatalyst (16,0)]
+		[iOS (16,0), Mac (13,0), MacCatalyst (16,0)]
```
Comment (tj-devel709) [inline src/corespotlight.cs:97]: Why this change? I don't see it in the todo files?

Comment (mandel-macaque) [reply src/corespotlight.cs:97]: The class already has a NoTV
